rllib/execution/train_ops.py

```diff
@@ -147,40 +179,10 @@ def multi_gpu_train_one_step(trainer, train_batch) -> Dict:
     trainer._counters[NUM_ENV_STEPS_TRAINED] += train_batch.count
     trainer._counters[NUM_AGENT_STEPS_TRAINED] += train_batch.agent_steps()
 
-    # Update weights - after learning on the local worker - on all remote
-    # workers.
-    if workers.remote_workers():
-        with trainer._timers[WORKER_UPDATE_TIMER]:
-            weights = ray.put(
-                local_worker.get_weights(
-                    local_worker.get_policies_to_train(train_batch)
-                )
-            )
-            for e in workers.remote_workers():
-                e.set_weights.remote(weights)
-
     return learner_info
 
 
 class TrainOneStep:
-    """Callable that improves the policy and updates workers.
-
-    This should be used with the .for_each() operator. A tuple of the input
-    and learner stats will be returned.
-
-    Examples:
-        >>> from ray.rllib.execution import ParallelRollouts
-        >>> workers = ... # doctest: +SKIP
-        >>> rollouts = ParallelRollouts(...) # doctest: +SKIP
-        >>> train_op = rollouts.for_each(TrainOneStep(workers)) # doctest: +SKIP
-        >>> # This trains the policy on one batch.
-        >>> print(next(train_op)) # doctest: +SKIP
-        SampleBatch(...), {"learner_stats": ...}
-
-    Updates the STEPS_TRAINED_COUNTER counter and LEARNER_INFO field in the
-    local iterator context.
-    """
-
```

Comment:
Just our of curiosity, why are we removing these docstrings?

Reply:
Ah, sorry, b/c we are starting to deprecate things and very soon would like users to only use the `synchronous_parallel_sample` and `asynchronous_parallel_requests` functions, instead.